forge/ee/routes/teamBroker/3rdPartyBroker.js

```diff
@@ -456,22 +450,32 @@ module.exports = async function (app) {
             // Get the placeholder creds object id used for team brokers
             brokerId = app.settings.get('team:broker:creds')
         }
-        const topics = Object.keys(request.body)
-        topics.forEach(async topic => {
-            const type = request.body[topic].type
-            try {
-                await app.db.models.MQTTTopicSchema.upsert({
-                    topic,
+        let body = request.body
+        if (!Array.isArray(body)) {
+            body = [body]
+        }
+        body.forEach(async topicInfo => {
+            if (topicInfo.topic) {
+                const topicObj = {
+                    topic: topicInfo.topic,
                     BrokerCredentialsId: brokerId,
-                    TeamId: teamId,
-                    inferredSchema: JSON.stringify(type)
-                }, {
-                    fields: ['inferredSchema'],
-                    conflictFields: ['topic', 'TeamId', 'BrokerCredentialsId']
-                })
-            } catch (err) {
-                // reply.status(500).send({ error: 'unknown_erorr', message: err.toString() })
-                // return
+                    TeamId: teamId
+                }
+                if (Object.hasOwn(topicInfo, 'type')) {
+                    topicObj.type = JSON.stringify(topicInfo.type)
```

Comment:
```suggestion
                    topicObj.inferredSchema = JSON.stringify(topicInfo.type)
```

This needs to match the column name (I didn't call it type in the db, because it is inferred.